From fa9f1f1f632bd060e124a401388130d1123813d6 Mon Sep 17 00:00:00 2001
From: Markos Chandras <markos.chandras@imgtec.com>
Date: Mon, 24 Jun 2013 14:42:21 +0000
Subject: [PATCH 020/105] MIPS: SNI: pcimt: Guard sni_controller with CONFIG_PCI

Fixes the following build problem:

arch/mips/sni/pcimt.c:188:30: error: 'sni_controller'
defined but not used [-Werror=unused-variable]

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
Acked-by: Steven J. Hill <Steven.Hill@imgtec.com>
Cc: linux-mips@linux-mips.org
Patchwork: https://patchwork.linux-mips.org/patch/5547/
Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
(cherry picked from commit 5bd807659bfd5d5a393b2269dc3e9699dddd9ad6)
---
 arch/mips/sni/pcimt.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/mips/sni/pcimt.c b/arch/mips/sni/pcimt.c
index cec4b8c..12336c2 100644
--- a/arch/mips/sni/pcimt.c
+++ b/arch/mips/sni/pcimt.c
@@ -185,6 +185,7 @@ static void __init sni_pcimt_resource_init(void)
 
 extern struct pci_ops sni_pcimt_ops;
 
+#ifdef CONFIG_PCI
 static struct pci_controller sni_controller = {
 	.pci_ops	= &sni_pcimt_ops,
 	.mem_resource	= &sni_mem_resource,
@@ -193,6 +194,7 @@ static struct pci_controller sni_controller = {
 	.io_offset	= 0x00000000UL,
 	.io_map_base	= SNI_PORT_BASE
 };
+#endif
 
 static void enable_pcimt_irq(struct irq_data *d)
 {
-- 
1.7.1

